connector/connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -165,4 +166,9 @@ message Expression {
   message QualifiedAttribute {
     string name = 1;
   }
+
+  message Alias {
```

Comment:
why wouldn't this be a common attribute of Expression instead?

Reply:
Only a few places require Expression to be wrapped by an alias. I'd like to match catalyst and have an individual `Alias` expression.